configure.ac

```diff
@@ -108,6 +108,15 @@ AC_DEFINE([BUILDOPT_HAVE_RPM_FILETRIGGERS], 1, [Set to 1 if we have file trigger
 AC_DEFINE([BUILDOPT_HAVE_RPMFILES], 1, [Set to 1 if we have rpmfiles API]),
 ], [have_modern_rpm=false])
 
+# https://github.com/projectatomic/rpm-ostree/pull/1213
+# You need to pass these so the jigdo side can convert from arch -> isa
+AC_ARG_WITH(rpm-isa,
+AS_HELP_STRING([--with-rpm-isa], [RPM's notion of _isa macro]),
+  AC_DEFINE_UNQUOTED([RPMOSTREE_RPM_ISA], ["${with_rpm_isa}"], [RPM isa]))
+AC_ARG_WITH(rpm-arch,
+  AS_HELP_STRING([--with-rpm-arch], [RPM's notion of _arch macro]),
+  AC_DEFINE_UNQUOTED([RPMOSTREE_RPM_ARCH], ["${with_rpm_arch}"], [RPM arch]))
```

Comment:
Let's default to `rpm -E %{_isa}` and `rpm -E %{_arch}` here? At least to make rojig Just Work for the machine users are playing on.

Reply:
Mmm.  Won't work when cross building though, and I do want e.g. [OpenEmbedded](http://www.openembedded.org/wiki/Main_Page) to be able to use rpm-ostree too.

But eh, we can figure that out down the line.